clean-docker.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+docker stop $(docker ps -a -q)
+docker rm $(docker ps -a -q)
```

Comment:
Personal utils should be placed out of the repo  (ie  ~/bin  or /usr/local/bin)

Reply:
true xD It was a commit where I just pushed everything in (plus it is no more needed now with the scripts)